chainer/links/caffe/caffe_function.py

```diff
@@ -221,7 +220,7 @@ def forward(self, inputs, outputs, disable=(), **kwargs):
             func = self.forwards[func_name]
             input_vars = tuple(variables[blob] for blob in bottom)
             output_vars = func(*input_vars)
-            if not isinstance(output_vars, collections_abc.Iterable):
+            if not isinstance(output_vars, tuple):
```

Comment:
I guess it's safer to use `(tuple, list)` here, considering issues like #1624 exists.